SVETLANA_SAT/classes/salad.js

```diff
@@ -0,0 +1,27 @@
+// Salad constructor
+function Salad(name, count, calories, price) {
+    this.name = name;
+    this.count = count;
+    this.calories = calories;
+    this.price = price;
```

Comment:
@Lana2683 We can create common class like Food or smth like that and move all these settings there, then call that common class by context of Salad or Drink or whatever.